### ISSUE TITLE
Reviewing and collecting statements for 1.1

### ISSUE THREAD

**Issue by ds10** [jiscdev/xapi#192]
On my list for next weeks call was to prepare the following in the 1.1 development branch. We are waiting on the 1.0 branch outputs from plugins so don't have a 1.1 branch, so I am leaving them as an issue tagged to discuss at the next call simply because the statements are currently spread over feature branches. They will be easier to review when they are merged into a 1.1 dev branch.

- VLE Quiz templates:

There are two templates in the feature quiz branch, that I think were almost completed and need checking and signing off, detail;s in ticket #188:

1. Answered questions - https://github.com/jiscdev/xapi/blob/feature-quiz/recipes/vle/answered_questions.md

2.  quiz completed - https://github.com/jiscdev/xapi/blob/feature-quiz/recipes/vle/quiz_completed.md

- VLE Forum templates

There is one forum template that has an if-this-then-that style to it, discussed in ticket #132 and living are in the VLE branch:

- https://github.com/jiscdev/xapi/blob/feature-vle/recipes/vle/forum.md

Again, I think this is at, check and sign off stage.

- Library templates

I have been working with Lee on library templates, these are in ticket #150 and sitting in the feature-library branch:

We have been working on a few, those that I think are close to completion and might make it to 1.1 are:

1. Borrowed: https://github.com/jiscdev/xapi/blob/feature-library/recipes/library/borrowing.md
2. e-content viewed: https://github.com/jiscdev/xapi/blob/feature-library/recipes/library/econtent.md
3. reading list item view: https://github.com/jiscdev/xapi/blob/feature-library/recipes/library/reading-list-view.md
4. reading list view: https://github.com/jiscdev/xapi/blob/feature-library/recipes/library/reading-list-item-view.md

There ae still some discussions hanging on these, Lee will correct me if I am wrong, but I think we left some details until we had a second data source? These are in a draft format, but have had quite a bit of discussion and exploration, particuarly the econtent viewing and borrowing statements
